apps/frontend/src/app/Formula/index.ts

```diff
@@ -532,6 +533,14 @@ const common: Data = {
         sum(one, prod(-1, enemy.defIgn))
       )
     ),
+    transDef: frac(
+      99999999,
+      prod(
```

Comment:
Shouldn't this be using the calculated `def` instead of being calculating again?

Reply:
I don't see any mathematical way to use the other def term in here